server/src-lib/Hasura/GraphQL/Execute.hs

```diff
@@ -182,41 +166,63 @@ getResolvedExecPlan
   -> SchemaCache
   -> SchemaCacheVer
   -> GQLReqUnparsed
-  -> m ExecPlanResolved
-getResolvedExecPlan pgExecCtx planCache userInfo sqlGenCtx
-  enableAL sc scVer reqUnparsed = do
-  planM <- liftIO $ EP.getPlan scVer (userRole userInfo)
-           opNameM queryStr planCache
+  -> m (Seq.Seq GQFieldResolvedPlan)
```

Comment:
In RJ, this will change to `GQFieldResolvedPlan` -> `QExecPlan` (https://github.com/hasura/graphql-engine/blob/821c1c3525e36f462bedee96908646e43225c442/server/src-lib/Hasura/GraphQL/Execute.hs#L180)

where QExecPlan is `type QExecPlan = ([QExecPlanUnresolved], GQFieldResolvedPlan)` (https://github.com/hasura/graphql-engine/blob/821c1c3525e36f462bedee96908646e43225c442/server/src-lib/Hasura/GraphQL/Execute.hs#L95)

Might be a good idea to rename `QExecPlan -> QFieldPlan` and `QExecPlanUnresolved -> GQRemoteRelUnresolvedPlan`